### PR TITLE
Hk network private network

### PR DESCRIPTION
**Pull request by KebirHakim**
How to share a private network between 2 public cloud projects

### CODE REVIEW

File: pages/network/Share Private Network/guide.en-gb.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+title: Network - How to share a private network between 2 public cloud projects
+excerpt: Learn how to share a private network between two public cloud projects on OVH Cloud
+updated: 2024-06-20
+---
+
+## Objective
+
+This guide will show you how to connect two projects on an internal network. It allows you to synchronize production and preproduction environments across several projects.
+
+## Capabilities and limitations
```
Comment (OVH-Team-Guides) [inline pages/network/Share Private Network/guide.en-gb.md]: This section is either empty and needs to be poupulated OR should be deleted

File: pages/network/Share Private Network/guide.en-gb.md [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+
+### Prerequisites
```
Comment (OVH-Team-Guides) [inline pages/network/Share Private Network/guide.en-gb.md]: Wrong term (Requirements) and markdown hierarchy (use an H2)

File: pages/network/Share Private Network/guide.en-gb.md [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+
+- Have 2 or more projects.
+
+### Procedure
```
Comment (OVH-Team-Guides) [inline pages/network/Share Private Network/guide.en-gb.md]: Wrong term (Instructions) and markdown hierarchy (use an H2 then adapt your hierarchy from it)

File: pages/network/Share Private Network/guide.en-gb.md [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+
+#### On our first project
```
Comment (OVH-Team-Guides) [inline pages/network/Share Private Network/guide.en-gb.md]: To rephrase

File: pages/network/Share Private Network/guide.en-gb.md [hunk 5]
```diff
@@ -0,0 +1,114 @@
...
+
+1. List the project IDs:
```
Comment (OVH-Team-Guides) [inline pages/network/Share Private Network/guide.en-gb.md]: Please skip a line after/before each code snippet

File: pages/network/Share Private Network/guide.en-gb.md [hunk 6]
```diff
@@ -0,0 +1,114 @@
...
+export OS_PROJECT_NAME=project_B
+```
+
+Or using the project ID:
+
+```sh
+export OS_PROJECT_ID=1234567890abcdef
+```
+
+On our second (or more) project :
```
Comment (OVH-Team-Guides) [inline pages/network/Share Private Network/guide.en-gb.md]: missing markdown title

**Review by OVH-Team-Guides — CHANGES_REQUESTED** [review]
A guide folder can only contain files named as `guide.language.md` so please delete the `Share_Private_Network` file or rename it.

The guide folder name contains empty spaces, which can't be accepted.

As this is a new guide, you should create a meta.yaml file and edit the index file.

You also have to write the fr-fr version of the guide.

**Comment by KebirHakim** [comment]
i create a new PR

**Pull request closed without merging**